src/styles/blocks/navigation.scss

```
@@ -0,0 +1,50 @@
+.nav {
+
+  &__list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    display: block;
+    line-height: 60px;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    height: 60px;
+    text-align: center;
+    color: $text-accent-color;
+
+    &:hover {
+      color: $main-color;
+    }
+  }
+}
+
+.is-active {
+  position: relative;
+  color: $main-color;
+
+  &::after {
+    position: absolute;
+    bottom: 0;
+    display: block;
+    height: 4px;
+    width: 100%;
+    content: " ";
```

не уверен влияет ли на что-то этот пробел, но я бы его убрал
```suggestion
    content: "";
```